Dockerfile

```diff
@@ -138,6 +139,32 @@ RUN echo 'Adding esp8266 toolchain' >&2 && \
 
 ENV PATH $PATH:/opt/esp/esp-open-sdk/xtensa-lx106-elf/bin
 
+# Install ESP32 toolchain in /opt/esp (181 MB after cleanup)
+RUN echo 'Installing ESP32 toolchain' >&2 && \
+    mkdir -p /opt/esp && \
+    cd /opt/esp && \
+    git clone --recursive https://github.com/espressif/esp-idf.git && \
```

Comment:
Is there a reason to not update to https://github.com/espressif/esp-idf/releases/tag/v3.1 now that it is released ?

Reply:
Thanks for the precision, so yeah there is a reason to stay with is one for the moment.